_posts/2019-02-30-nlog-4-6-is-live.md

```diff
@@ -0,0 +1,227 @@
+---
+layout: post
+title: NLog 4.6 is live!
+---
+
+NLog 4.6 contains lots of new features and performance improvements.
+
+## Main features
+
+### DatabaseTarget supports parameter types
+It is now possible to configure the type for the DatabaseTarget parameters.
+Instead of always sending string-value then one can write the actual value-type.
+
+```xml
+<target name="db" xsi:type="Database" connectionstring="..." >
+  <commandtext>INSERT INTO FooBar VALUES(@ts, @lvl, @msg)</commandtext>
+  <parameter name="@ts" layout="${date}" dbType="SqlDbType.DateTime2" />
+  <parameter name="@lvl" layout="${level}" dbType="DbType.Int32" />
+  <parameter name="@msg" layout="${message}" dbType="SqlDbType.VarChar" size="-1" parameterType="String" />
+</target>
+```
+
+When using a non-string DbType then it will attempt to convert the raw value directly to the expected type.
+When using special DbType like `SqlDbType.SmallInt` then one can specify `parameterType="Int16"` to help.
```

Comment:
I think this is indeed important, but maybe it should be only in the wiki and not in the release post. (it is very much in-depth). Is that OK with you?

Reply:
Good idea. Think I was a little too happy about the change :)